rsmtool/reader.py

```diff
@@ -239,7 +239,7 @@ def read_from_file(filename, converters=None, **kwargs):
         if file_extension in ['.csv', '.tsv']:
             sep = '\t' if file_extension == '.tsv' else ','
             do_read = partial(pd.read_csv, sep=sep, converters=converters)
-        elif file_extension in ['.xls', '.xlsx']:
+        elif file_extension == '.xlsx':
             do_read = partial(pd.read_excel, converters=converters)
         elif file_extension in ['.sas7bdat']:
```

Comment:
```suggestion
        elif file_extension == '.sas7bdat':
```

Just to be consistent?